scrapers/pjgirls.yml

```
@@ -0,0 +1,37 @@
+name: pjgirls
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pjgirls.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //div[@class='container']/h1
+        postProcess:
+          - replace:
+              - regex: "VIDEO:"
+                with: ""
+     Date:
+        selector: //div[@class='info']/h3[1]
```

From a quick look it doesnt appear that there is a video without a date
Just to be sure though
```suggestion
        selector: //div[@class='info']/h3[not(starts-with(.,"LENGTH"))]
```